CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [1.31](https://github.com/brave/brave-ios/releases/tag/v1.31)
+
+- Added auto-play settings for sites to auto-play video/audio. ([#1738](https://github.com/brave/brave-ios/issues/1738))
+- Added "Scan QR Code" shortcut when long pressed on Brave on home-screen. ([#3798](https://github.com/brave/brave-ios/issues/3798))
+- Added new playlist URL button and menu notification icon. ([#3976](https://github.com/brave/brave-ios/issues/3976))
+- Added support for background playback and picture-in-picture (PiP) support for videos. ([#4031](https://github.com/brave/brave-ios/issues/4031))
+- Updated "Brave Search beta" to "Brave Search" under search engines and on the on-boarding screen. ([#4049](https://github.com/brave/brave-ios/issues/4049))
+- Updated settings for playlist menu notification. ([#4098](https://github.com/brave/brave-ios/issues/4098))
+- Added support for IPv6 address resolution. ([#3866](https://github.com/brave/brave-ios/issues/3866))
+- Fixed preview URLs always showing in lowercase. ([#3901](https://github.com/brave/brave-ios/issues/3901))
+- Fixed shields stats not updating in certain cases. ([#4021](https://github.com/brave/brave-ios/issues/4021))
+- Fixed share menu only sharing the canonical (base) URL instead of the actual page URL. ([#4071](https://github.com/brave/brave-ios/issues/4071))
+- Fixed Brave crashing in certain cases when history sync is enabled. ([#4073](https://github.com/brave/brave-ios/issues/4073))
+- Fixed issue when returning search results instead of computing arithmetic queries typed into the URL bar. ([#4081](https://github.com/brave/brave-ios/issues/4081))
+- Fixed links not marked as visited when revisiting a search link via "Open in Brave". ([#4082](https://github.com/brave/brave-ios/issues/4082))
+- Fixed issue where audio continues to play on tab when opening and closing the 3-dot menu. ([#4092](https://github.com/brave/brave-ios/issues/4092))
```

Comment:
we should use the correct name for the menu rather than `3-dot menu`. I'm not really sure what it's officially called. We've used `hamburger` or `kabob` menu's in the past. I've used `Settings` menu before. Maybe @aekeus knows what the correct wording should be.

Reply:
We have referred to that menu as “…” (more) in the AppStore notes before

Reply:
some how more menu doesn't sound right in this context. so just going to make it settings menu